addon/mixins/taggable-mixin.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Mixin.create({
             Ember.A(currentTags);
```

Comment:
I think I still need the slice or something similar - I need to make a copy that doesn't reference the original in order for the changes to persist.